### DISCUSSION TITLE
Where is PowerOnState?

### DISCUSSION THREAD

**Discussion by Marmachine** [arendst/Tasmota#20898] [General]
I am new to Tasmota, however i have flashed a WemosD1 mini and want to control an 8ch Relay module. So far, so good. I toggle them with a curl command as well as through web-ui.

My issue is that, by default all relays are activated and switching them to On state actually deactivates the relays. How can i reverse this? I read about PowerOnState but can't find that in config.

**Comment by barbudor** [comment]
More likely your template is wrong and you should use Relay_i (inverted) instead of Relay (or the other way round)

**Reply by barbudor** [reply]
And more generally you should look for Commands
https://tasmota.github.io/docs/Commands/

In Tasmota, 80% is done through commands vs Web UI

**Reply by Marmachine** [reply]
Thx... that was helpfull!
